core/block_svg.js

```diff
@@ -1242,16 +1242,14 @@ Blockly.BlockSvg.prototype.setHighlighted = function(highlighted) {
  * Select this block.  Highlight it visually.
  */
 Blockly.BlockSvg.prototype.addSelect = function() {
-  Blockly.utils.dom.addClass(
```

Comment:
Might want to think about whether we keep this as a "framework" level task, and let each renderer actually implement the details of what rendering selected looks like.